packages/flutter/test/material/theme_test.dart

```diff
@@ -254,13 +255,13 @@ void main() {
     );
     await tester.pump(const Duration(milliseconds: 100)); // Halfway through the theme transition
 
-    glyphText = tester.renderObject(find.byType(RichText));
+    glyphText = tester.renderObject(find.byType(RichText, skipOffstage: false));
```

Comment:
This actually doesn't change the test, but instead attempts to keep it as is. The test tests something that contains `ExcludeSemantics`, which changes its behavior in this PR. Because the tests wants to scan offstage content it needed an update.